flat_mutation_reader.hh

```diff
@@ -61,6 +61,11 @@ GCC6_CONCEPT(
     concept bool PartitionFilter = requires(T filter, const dht::decorated_key& dk) {
         { filter(dk) } -> bool;
     };
+
+    template<typename T>
+    concept bool MutationFilter = requires(T filter, const mutation_fragment& mf) {
```

Comment:
This looks like a `MutationFragmentFilter`. `Mutation` represents a partition.

Reply:
Ok. Will change.